app/models/manageiq/providers/azure/cloud_manager/metrics_capture.rb

```diff
@@ -1,231 +1,225 @@
+  private_constant *%i[INTERVAL_1_MINUTE VIM_INTERVAL]
 
-  COUNTER_INFO = [
+  COUNTERS_INFO = [
+    ## CPU
     {
-      :native_counters       => CPU_METERS,
-      :calculation           => ->(stat) { stat.first },
-      :vim_style_counter_key => "cpu_usage_rate_average"
+      :counter_key => 'cpu_usage_rate_average',
+      :source      => :api,
+      :metrics     => [
+        'Percentage CPU',
+      ].freeze,
+      :calculation => ->(stats) { stats.mean },
+      :unit_key    => 'percent',
     },
+    ## Memory
     {
-      :native_counters       => NETWORK_METERS,
-      :calculation           => ->(stat) { stat.first / 1.kilobyte },
-      :vim_style_counter_key => "net_usage_rate_average",
+      :counter_key => 'mem_usage_rate_average',
+      :source      => :raw,
+      :metrics     => [
+        '/builtin/memory/percentusedmemory', # linux
+        '\Memory\% Committed Bytes In Use', # windows
+      ].freeze,
+      :calculation => ->(stats) { stats.mean },
+      :unit_key    => 'percent',
     },
+    ## Disk
     {
-      :native_counters       => MEMORY_METERS,
-      :calculation           => ->(stat) { stat.first },
-      :vim_style_counter_key => "mem_usage_absolute_average",
+      :counter_key => 'disk_usage_absolute_average',
+      :source      => :api,
+      :metrics     => [
+        'Per Disk Read Bytes/sec',
+        'Per Disk Write Bytes/sec',
+      ].freeze,
+      :calculation => ->(stats) { stats.sum / 1.kilobyte },
+      :unit_key    => 'kilobytespersecond',
     },
+    ## Network
     {
-      :native_counters       => DISK_METERS,
-      :calculation           => ->(stat) { stat.sum / 1.kilobyte },
-      :vim_style_counter_key => "disk_usage_rate_average",
-    }
-  ].freeze
-
-  COUNTER_NAMES = COUNTER_INFO.flat_map { |i| i[:native_counters] }.uniq.to_set.freeze
-
-  VIM_STYLE_COUNTERS = {
-    "cpu_usage_rate_average"     => {
-      :counter_key           => "cpu_usage_rate_average",
-      :instance              => "",
-      :capture_interval      => "20",
-      :precision             => 1,
-      :rollup                => "average",
-      :unit_key              => "percent",
-      :capture_interval_name => "realtime"
-    },
-    "mem_usage_absolute_average" => {
-      :counter_key           => "mem_usage_absolute_average",
-      :instance              => "",
-      :capture_interval      => "20",
-      :precision             => 1,
-      :rollup                => "average",
-      :unit_key              => "percent",
-      :capture_interval_name => "realtime"
-    },
-    "net_usage_rate_average"     => {
-      :counter_key           => "net_usage_rate_average",
-      :instance              => "",
-      :capture_interval      => "20",
-      :precision             => 1,
-      :rollup                => "average",
-      :unit_key              => "kilobytespersecond",
-      :capture_interval_name => "realtime"
+      :counter_key => 'net_usage_absolute_average',
```

Comment:
As per our gitter discussion I think it has to be `net_usage_rate_average` to match the column in the `metrics` table. Same for disk usage.

Reply:
done